site/app/templates/officeHoursQueue/NewQueue.twig

```diff
@@ -7,10 +7,11 @@
         <input type="hidden" name="csrf_token" value="{{ csrf_token }}"/>
         <input type="text" id="new_queue_code" name="code" placeholder="New Queue Name" aria-label="New Queue Name" required="required" maxlength="2048" style="margin-top: 1rem;"><br>
         <div class="access-code-input" style="margin-top: 1rem;">
-            <input type="text" id="new_queue_token" name="token" placeholder="Access Code" aria-label="Access Code" required="required" maxlength="2048">
+            <input type="text" id="new_queue_token" name="token" placeholder="Access Code" aria-label="Access Code" maxlength="2048">
```

Comment:
HTML Ids should use dashes not underscores 
https://submitty.org/developer/coding_style_guide/css